src/_guides/libraries/objective-c-interop.md

```diff
@@ -328,6 +341,34 @@ to the correct isolate.
 For an example of this,
 see the implementation of [`package:cupertino_http`][].
 
+There are a few steps required to handle callbacks:
+
+1. Create an Objective-C function, or method, 
+that calls the method of interest. Your helper function or method 
+must include a callback that transforms the returned data 
+into Dart C Objects and sends the data, or a pointer to the object, 
+to the Dart Port. You can see an [example here](https://github.com/dart-lang/http/blob/2f8205c5254886e088aa5ebd52a2e4dc4ec18afb/pkgs/cupertino_http/src/CUPHTTPClientDelegate.m#L63).
```

Comment:
I'm a bit queasy about linking a random git hash version. Does `dart-lang/http` tag releases, or should we link to `master`/`main` with the understanding there will be maintenance overhead?

Reply:
I will let @brianquinlan chime in on this question